src/plugins/strapi-plugin-internal-links/server/services/external-source.ts

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+
+type externalApiData = {
+	url: string;
+	searchQuery: string;
+};
+
+const getexternalApiData = async (data: externalApiData) => {
+	const url = createUrl(data);
```

Comment:
in de admin wordt de url als optioneel beschouwd. Dus hier ook ff een check doen of die bestaat.